manifests/nodejs.yml

```
@@ -1251,7 +1251,7 @@ tests/:
       Test_Config_TraceEnabled: *ref_4_3_0
       Test_Config_TraceLogDirectory: missing_feature
       Test_Config_UnifiedServiceTagging: *ref_5_25_0
-      Test_Stable_Config_Default: *ref_5_27_0
+      Test_Stable_Config_Default: irrelevant (temporarily ignored)
```

```suggestion
      Test_Stable_Config_Default: missing_feature (unignore in 5.62.0)
```

Looks good, just figured we should not use irrelevant to not accidentally leave this as untested after we make the relevant changes 

Lmk what you think